scripts/portstat

```diff
@@ -27,537 +21,25 @@ try:
         sys.path.insert(0, modules_path)
         sys.path.insert(0, tests_path)
         import mock_tables.dbconnector
+
+        if os.environ["UTILITIES_UNIT_TESTING_IS_SUP"] == "1":
```

Comment:
seems the most useful change is here.
other changes are some refactoring and tests

Reply:
No, the most useful change is the addition of collect_stat_from_lc in utilities_common/portstat.py.

Reply:
sorry was confused, where is get_cnstat_dict used

Reply:
get_cnstat_dict is used by script/portstat to collect counters.